client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.html

```diff
@@ -0,0 +1,84 @@
+<div class="widget-presentation">
+
+    <h2 class="widget-presentation-title" data-translate="xdsl_access_comfort_exchange_title"></h2>
+
+    <tuc-toast-message></tuc-toast-message>
+
+    <div data-ng-if="$ctrl.isRMAOpened" class="opened-rma">
+        <div class="rma-line" data-translate="xdsl_access_comfort_exchange_already_opened_rma"></div>
+        <div data-ng-repeat="rma in $ctrl.rmas track by $index">
+            <div class="rma-line">
+                <span class="rma-detail-label col-sm-3" data-translate="xdsl_access_comfort_exchange_rma_status"></span>
```

Comment:
You cannot use a `col-sm-3` without having a parent `.row` element.

ref: https://github.com/twbs/bootlint/wiki/E014